### DISCUSSION TITLE
Python Supabase Module?

### DISCUSSION THREAD

**Discussion by Falcn8** [supabase/supabase#4976] [Questions] [answered]
Is there a Python Supabase module? I want to upload files to Supabase and retrieve files but I can't find it in the docs.

**Comment by inian** [comment, marked as answer]
We don't have a python module for storage yet, you can find the full list of community supported modules [here](https://github.com/supabase-community)

**Comment by Falcn8** [comment]
Ok, I understand. Thank you.
